kubernetes/jobmonitor/src/main/kotlin/LostJobsFinder.kt

```diff
@@ -51,6 +55,10 @@ import org.slf4j.MDC
  * status in Kubernetes. It looks for jobs that should be running according to the database, but do not exist
  * (anymore) in Kubernetes. For such jobs, it sends a notification to the Orchestrator, to give it the chance to act
  * accordingly.
+ *
+ * Another problem detected by this class is missing schedules, i.e. active ORT runs for which all jobs have been
+ * completed. For such runs, a different type of notification is sent to the Orchestrator, so that it try to resume
```

Comment:
it _tries_

Reply:
Actually, it should say "it can try". Changed accordingly.